# HG changeset patch
# User Rik <rik@octave.org>
# Date 1449418769 28800
#      Sun Dec 06 08:19:29 2015 -0800
# Node ID 6935b8f038cc5e04aa5ce5279946a5a6fc6d08a4
# Parent  2da4058d65c758c622f86179704bec874f8fd10c
Fix BIST tests for rows()

* data.cc (Frows): Fix BIST tests.

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -2918,17 +2918,17 @@ Return the number of rows of @var{a}.\n\
 %! x(2,5,3).a = 1;
 %! assert (rows (x), 2);
 %! y(5,4,3).b = 2;
 %! assert (rows (y), 5);
 
 %!assert (rows ("Hello World"), 1)
 
 %!assert (rows ([]), 0)
-%!assert (rows (zeros (2,0), 2))
+%!assert (rows (zeros (2,0)), 2)
 
 ## Test input validation
 %!error rows ()
 %!error rows (1,2)
 */
 
 DEFUN (columns, args, ,
        "-*- texinfo -*-\n\
